app/views/cost_of_living_landing_page/show.html.erb

```diff
@@ -28,20 +28,25 @@
   end
 %>
 
+<%= render "shared/browse_breadcrumbs", {
+  breadcrumbs: breadcrumbs,
+  column_two_thirds: true,
+  collapse_on_mobile: true,
+  inverse: true
+} %>
+
 <header class="cost-of-living-hub__header">
```

Comment:
I think we can now delete the `cost-of-living-hub__header` class from this wrapper, seeing that we removed the related style for it. 

(In fact, I think we don't need the `<header>` tag here, we don't have it on the browse pages. That's unrelated to the changes in this PR but we could just remove the wrapper here too.)

Reply:
Good spot, thank you. I've now removed not only the header but also all the redundant row/column classes, and adjusted the margins to match the browse pages. This is now much cleaner and more robust.

Reply:
Good spot, thank you. I've now removed the header element and also various redundant rows/columns, and adjusted margins accordingly to match the treatment used on Browse pages.